docs/getting-started/devenv/using-visual-studio-code.md

```diff
@@ -1,16 +1,381 @@
-> 
-> If you would like to review early drafts and outlines of this
-> topic, please leave a note with your contact information in the issue.
->
-> Learn more about how you can contribute on
-> [GitHub](https://github.com/dotnet/core-docs/blob/master/CONTRIBUTING.md).
->
+# Development using Visual Studio Code
+
+by [Bertrand Le Roy](https://github.com/bleroy),  [Phillip Carter](https://github.com/cartermp),
+[Bill Wagner](https://github.com/billwagner)
+
+Contributions by [Toni Solarin-Sodara](https://github.com/tsolarin)
+
+This document provides a tour of the steps and workflow to create a .NET
+Core Solution using [Visual Studio Code](http://codde.visualstudio.com).
+You'll learn how to create projects, create unit tests, use the debugging
+tools, and incorproate third-party libraries via [NuGet](http://nuget.org).
+
+This article uses Visual Studio Code on Mac OS. Where there are differences,
+it points out the differences for the Windows platform.
+
+Prerequisites
+-------------
+
+Before starting, you'll need to install the [.NET Core SDK](https://www.microsoft.com/net/download),
+currently in a preview relese. The .NET Core SDK includes the latest release
+of the .NET Core framework and runtime.
+
+You'll also need to install [Visual Studio Code](http://code.visualstudio.com).
+During the course of this article, you'll also install extensions
+that will improve the .NET Core development experience.
+
+You can find the links to all of these at the [.NET home page](http://dot.net).
+
+Getting Started
+---------------
+
+The source for this tutorial is available on
+[GitHub](https://github.com/dotnet/core-docs/tree/master/samples/getting-started/golden).
+
+Start Visual Studio Code. Press Ctrl + '`' (the back-quote character) to open
+an embedded terminal in VS Code. (Alternatively, you can use a separate
+terminal window, if you prefer).
+
+By the time we're done, you'll create three projects: a library project,
+tests for that library project, and a console application that makes
+use of the library. You'll follow a standard folder structure for
+the three projects. Following this standard folder structure
+maeans that the .NET Core SDK tools understand the relationship between
+your production code projects and your test code projects. That makes
+your development exerience more productive.
+
+Let's start by creating those folders. In the terminal, create a 'golden'
+directory. Under that directory create `src` and `test`
+directories. Under `src` create `app` and `library` directories. In `test`
+create a `test-library` directory. You can do this either using the terminal
+in VS code, or by clicking on the parent folder in VS Code and selecting the
+"New Folder" icon.
+
+In VS Code, open the 'golden' directory. This directory is the root of your solution.
+
+Next, create a `global.json` file in the root directory for your solution.
+The contents of `global.json` are:
+
+```js
+{
+    projects: [
```

Comment:
In Windows "projects" shows up in red with an error message saying property keys must be double-quoted.